docker-compose.yml

```
@@ -1,14 +1,13 @@
 version: '3.7'
 
 services:
-  yarn:
+  bash:
     image: front-end-monorepo_dev:latest
     build:
       context: ./
       target: bootstrap
     entrypoint:
-      - "yarn"
-    command: ["test:ci"]
+      - "/bin/sh"
```

For clarity and to ensure we always get a `bash` shell i think we should be explicit here? 
https://superuser.com/questions/1133187/when-must-i-use-bin-bash-and-when-bin-sh
```suggestion
      - "/bin/bash"
```